package.json

```diff
@@ -51,8 +51,9 @@
   "homepage": "https://github.com/Automattic/vip#readme",
   "devDependencies": {
     "@babel/cli": "7.8.4",
-    "@babel/core": "7.9.0",
+    "@babel/core": "^7.11.1",
```

Comment:
This is not essential for this change and I'm happy to pull it out if someone finds a good reason.